src/forms/inputs/input.js

```diff
@@ -25,6 +25,8 @@ import {
  * @param {Object} input - A `redux-forms` [input](http://redux-form.com/6.5.0/docs/api/Field.md/#input-props) object
  * @param {Object} meta - A `redux-forms` [meta](http://redux-form.com/6.5.0/docs/api/Field.md/#meta-props) object
  * @param {String} [type] - A string to specify the type of input element (defaults to `text`)
+ * @param {String} [ariaLabel] - A string to specify the label of the input element to assistive technologies when a standard label is not provided (will be overriden by `ariaLabelledby`)
+ * @param {String} [ariaLabelledby] - A string that corresponds to the `id` of one or more elements used as the label of the input element for assistive technologies when a standard label is not provided (will override `ariaLabel`)
```

Comment:
@chawes13 Before I go too crazy with this (adding these to all the other `input` based components), does this look like the right pattern to follow? I don't see too many aria type props spelled out this way, but I wonder if we should be doing that.

Reply:
@danparnella I'm not sure I follow the need for this change. I can pass it straight through the `Field` component and it gets added correctly.

E.g., https://codesandbox.io/s/tender-marco-giusn?file=/src/Form.js

Reply:
🤔 Hopefully, I was just following what we had done in artlook: https://github.com/parliamentdata/artlook-client/pull/808/files#diff-431ff99478b73c952842cc3b24c9fe645eda70de5f5a98eb19cb472a64c3a694 and that we can just pass it through like you said. I'll go down that path and revert these changes!